src/ssl.c

```diff
@@ -36978,7 +36978,7 @@ int wolfSSL_BN_is_word(const WOLFSSL_BIGNUM* bn, WOLFSSL_BN_ULONG w)
         return WOLFSSL_FAILURE;
     }
 
-    if (w <= (WOLFSSL_BN_ULONG)MP_MASK) {
+    if (w <= MP_MASK) {
```

Comment:
This cast should be valid/correct. I don't recommend removing it. Same for other two.